packages/tools/src/tools/CrosshairsTool.ts

```diff
@@ -1288,17 +1289,18 @@ export default class CrosshairsTool extends AnnotationTool {
       }
     });
 
+    renderStatus = true;
+
     // Save new handles points in annotation
     data.handles.rotationPoints = newRtpoints;
     data.handles.slabThicknessPoints = newStpoints;
 
-    // render a circle to pin point the viewport color
-    // TODO: This should not be part of the tool, and definitely not part of the renderAnnotation loop
-
     if (!this.configuration.viewportIndicators) {
-      return;
+      return renderStatus;
```

Comment:
Wouldn't it be cleaner to just stick the reference color rendering inside:

`if (this.configuration.viewportIndicators) { ... }`

And then just have one `return renderStatus;` line at the end of the method?